docs/data/date-pickers/time-picker/time-picker.md

```diff
@@ -95,6 +95,13 @@ You can force a specific orientation using the `orientation` prop.
 
 {{"demo": "StaticTimePickerLandscape.js", "bg": true}}
 
+## Choose time view renderer
+
+You can use the `viewRenderers` prop to change the view that is used for rendering a view.
+You might be interested in using the Time Clock instead of the Digital Clock on desktop as well as mobile.
```

Comment:
How about the following?

```suggestion
You might be interested in using the [TimeClock](https://mui.com/x/react-date-pickers/time-clock/) instead of the Digital Clock on desktop as well as mobile.
```

Reply:
Updated all component names with links. 👍